gcp_variant_transforms/libs/infer_headers_util.py

```diff
@@ -25,15 +25,119 @@
 
 from gcp_variant_transforms.beam_io import vcf_header_io
 from gcp_variant_transforms.beam_io import vcfio  # pylint: disable=unused-import
-from gcp_variant_transforms.libs.annotation import annotation_parser
 from gcp_variant_transforms.libs import vcf_field_conflict_resolver
+from gcp_variant_transforms.libs.annotation import annotation_parser
 
 _FIELD_COUNT_ALTERNATE_ALLELE = 'A'
 
 # Alias for the header key/type constants to make referencing easier.
 _HeaderKeyConstants = vcf_header_io.VcfParserHeaderKeyConstants
 _HeaderTypeConstants = vcf_header_io.VcfHeaderFieldTypeConstants
 
+# Filled with annotation field and name data, then used as a header ID.
+_BASE_ANNOTATION_TYPE_KEY = '{}_{}_TYPE'
+
+def get_inferred_annotation_type_header_key(annot_field, name):
+  # type: (str, str) -> str
+  """Creates ID values for annotation type info headers.
+
+  Args:
+    annot_field: field name representing annotation field (e.g. 'CSQ').
+    name: annotation data field names (e.g. 'IMPACT').
+
+  Returns:
+    Info ID value (e.g. CSQ_IMPACT_TYPE).
+  """
+  return _BASE_ANNOTATION_TYPE_KEY.format(annot_field, name)
+
+def infer_info_fields(
+    variant,
```

Comment:
nit: Please add type for this variable and the one below.

Reply:
Done.